amundsen_application/static/js/components/AvatarLabel/index.tsx

```diff
@@ -11,24 +11,33 @@ export interface AvatarLabelProps {
   avatarClass?: string;
   labelClass?: string;
   label?: string;
+  round?: boolean;
   src?: string;
 }
 
 const AvatarLabel: React.FC<AvatarLabelProps> = ({
   avatarClass,
   labelClass,
   label,
+  round,
   src,
 }: AvatarLabelProps) => (
   <div className="avatar-label-component">
-    <Avatar className={avatarClass} name={label} src={src} size={24} round />
+    <Avatar
+      className={avatarClass}
+      name={label}
+      src={src}
+      size={24}
+      round={round}
+    />
     <span className={`avatar-label body-2 ${labelClass}`}>{label}</span>
   </div>
 );
 
 AvatarLabel.defaultProps = {
   labelClass: 'text-secondary',
   label: '',
+  round: true,
   src: '',
 };
```

Comment:
Would you mind to move this into the destructuring in lines 19-23?
We are going to deprecate using `.defaultProps` altogether, there isn't a good reason for them anymore.

Reply:
@Golodhros Fair enough. I've got this fixed in my local copy. Before pushing, can you help me work around this `eslint` error? 

https://github.com/amundsen-io/amundsenfrontendlibrary/pull/859/checks?check_run_id=1647122266#step:5:19

Reply:
Oh, that seems like a false positive we should fix. For now, just run `npm run betterer:update` to get the exception approved. 
I will try to look into it this week.

Thanks!